.travis.yml

```diff
@@ -0,0 +1,66 @@
+########################################################################
+## Travis CI config for LimeSuite
+##
+## * not all components will be configured
+## * the matrix builds Release and Debug
+## * run the self tests after install
+########################################################################
+
+sudo: required
+dist: trusty
+
+language: cpp
+
+compiler:
+  - clang
+  - gcc
+
+os:
+  - linux
+  - osx
+
+env:
+  global:
+    - INSTALL_PREFIX=/usr/local
+    - SOAPY_SDR_BRANCH=master
+  matrix:
+    - BUILD_TYPE=Debug
+    - BUILD_TYPE=Release
+
+# whitelist
+branches:
+  only:
+    - master
+    - stable
+
+before_install:
+  
+  
+  # update after package changes
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get -qq update; fi;
```

Comment:
Excellent. In the past I needed to supply a cmake 3 PPA. That is nice to hear that travis updated cmake by default.